app/views/katello/api/v2/subscriptions/base.json.rabl

```diff
@@ -0,0 +1,19 @@
+object @resource
+
+attributes :cp_id => :id
+
+# When attached candlepin entitlements are returned (eg. for subscriptions attached
+# to systems), the 'id' is the entitlement id. This field is for referencing the
+# original subscription.
+attributes :subscription_id
+attributes :description
+attributes :name
+attributes :start_date, :end_date
+attributes :available, :quantity, :consumed, :amount
+attributes :account_number, :contract_number
+attributes :support_type, :support_level
+attributes :product_id
+attributes :arch, :virt_only, :unmapped_guest
+attributes :sockets, :cores, :ram
+attributes :instance_multiplier, :stacking_id, :multi_entitlement
+attributes :type
```

Comment:
They were not, its slowing down showing the subscriptions/pools on index, I refactored and will push up soon